.vscode/tasks.json

```
@@ -0,0 +1,67 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Run Jekyll Server",
+      "type": "shell",
+      "command": "./tools/run.sh",
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "problemMatcher": [],
+      "detail": "Runs the Jekyll server with live reload."
+    },
+    {
+      "label": "Build Jekyll Site",
+      "type": "shell",
+      "command": "./tools/test.sh",
+      "group": {
+        "kind": "build"
+      },
+      "problemMatcher": [],
+      "detail": "Build the Jekyll site for production."
+    },
+    {
+      "label": "Build JS (watch)",
+      "type": "shell",
+      "command": "npm run watch:js",
+      "group": {
+        "kind": "build"
+      },
+      "problemMatcher": [],
+      "detail": "Build JS files in watch mode."
+    },
+    {
+      "label": "Build CSS",
+      "type": "shell",
+      "command": "npm run build:css",
+      "group": {
+        "kind": "build"
+      },
+      "problemMatcher": [],
+      "detail": "Build CSS files."
+    },
+    {
+      "label": "Build JS & CSS",
+      "type": "shell",
+      "command": "npm run build",
+      "group": {
+        "kind": "build"
+      },
+      "problemMatcher": [],
+      "detail": "Build JS & CSS for production."
+    },
+    {
+      "label": "Run Jekyll Server + Build JS (watch)",
+      "dependsOn": [
+        "Run Jekyll Server",
+        "Build JS (watch)"
+      ],
```

Are you using a custom vscode formatter settings locally? Why is the JSON formatting not the same?

```suggestion
      "dependsOn": ["Run Jekyll Server", "Build JS (watch)"],
```

---

> Are you using a custom vscode formatter settings locally?

I'm not. I work inside dev container, so it should be same.

> Why is the JSON formatting not the same?

I have no idea. Do you use dev container too?

---

Yes, all settings and extensions come from `.vscode` and `.devcontainer`.

---

Can you share the [user settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson)?

---

It seems the default Prettier behavior should be as you suggest, so I made requested change.

Thanks for your time on this. I didn't think it would take this long.

---

> Can you share the [user settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson)?

Okay, so my expectation was when you work inside devcontainer, local settings do not apply.

I had custom setting for `jsonc` there and after removing it I got what you would expect.

```json
"[json]": {
  "editor.defaultFormatter": "vscode.json-language-features"
},
```

The problem is that my `settings.json` is huge, so there may be other conflicting cases. That's pity. I wonder why my local settings have applied? Is this because local setting is more targeted in this case?